docs/modules/airflow/pages/usage-guide/security.adoc

```diff
@@ -1,14 +1,21 @@
 = Security
-:description: Secure Apache Airflow by configuring user authentication and authorization, either with built-in methods or LDAP.
+:description: Secure Apache Airflow by configuring user authentication and authorization.
 :airflow-access-control-docs: https://airflow.apache.org/docs/apache-airflow/stable/security/access-control.html
+:keycloak: https://www.keycloak.org/
 
 Secure Apache Airflow by configuring user authentication and authorization.
-Airflow provides built-in user and role management, but can also connect to a LDAP server to manage users centrally instead.
+Airflow provides built-in user and role management, but can also connect to an LDAP server or an OIDC provider to manage users centrally instead.
 
 == Authentication
 
-Users need to authenticate themselves before using Airflow, and there are two ways to configure users:
-The built-in user management or LDAP.
+Users need to authenticate themselves before using Airflow, and there are several ways in which this can be set up.
+￼
+[IMPORTANT]
+.Multiple authentication methods
```

Comment:
I see two login boxes in the UI, is that correct?
![image](https://github.com/user-attachments/assets/8018689d-a358-4326-9481-669308e40307)

Reply:
When I click on the first one, I get "This site can’t be reached" (DNS issue) and the second one doesn't respond. I also see a "Login" button top right, which does not respond. I think this will be confusing for users: can we correct and/or document this a bit more?

Reply:
Yes this is confusing. 

However in superset we face the same issue and yet it works as intended. If you want to suppress this behaviour, you can write your own `securitymanager.py`. Thus we can't correct it but the we offer the user to correct it on his own with https://github.com/stackabletech/airflow-operator/pull/493

Reply:
Don't we want/need to do that for when we use Airflow with the SSI stack?

Reply:
That's yet up to test but with superset we have to do it in order to use SSI, yes

Reply:
I still don't think we want to directly introduce it to the airflow image just yet as other user might not want it. However, we can leave an example somewhere in the docs ( might be we did that already )

Reply:
We can open a separate ticket to add a test or create a demo, but shouldn't we at least make sure in this PR that it works for external access, as there is where most users will be using the UI? At the moment we are only supporting in-cluster usage.

Reply:
As discussed, let's open a new issue for a demo, much like the superset one.